ts/project.ts

```diff
@@ -22,11 +22,45 @@ export type ProjectData = {
   defaultExecutable?: Executable;
 };
 
+/**
+ * `Plugin`s automatically add one or more fields to your project. You can use
+ * existing plugins to automatically add a few useful fields without having to
+ * declare them manually yourself.
+ *
+ * Here's an example of a simple `Plugin` and how to use it:
+ *
+ * ```typescript
+ * import * as garn from "https://garn.io/ts/v0.0.15/mod.ts";
+ *
+ * const prettier =
+ *   (
+ *     globPattern: string,
+ *   ): garn.Plugin<
+ *     { formatCheck: garn.Check; format: garn.Executable },
+ *     garn.ProjectHelpers
+ *   > =>
+ *   (p) =>
+ *     p
+ *       .withDevTools([
+ *         garn.mkPackage(garn.nix.nixRaw`pkgs.nodePackages.prettier`, "prettier"),
```

Comment:
I forget - what's the status of including `nodePackage` in nixpkgs? If it's merged, then this should be changed. If not, then reminder to change this (and the website!) when it is merged.

(Maybe cc @alexdavid)

Reply:
Yup, not merged yet. Here's the PR: #382.